src/api/l_thread.c

```diff
@@ -86,7 +86,9 @@ static int l_lovrThreadGetChannel(lua_State* L) {
   const char* name = luaL_checkstring(L, 1);
   Channel* channel = lovrThreadGetChannel(name);
   luax_pushtype(L, Channel, channel);
-  lovrRelease(Channel, channel);
+  // Note: Channels are intentionally not released here
+  // Because the ownership of Channels is different from most objects, it is simpler to clean them
+  // all up in lovrThreadModuleDestroy.
```

Comment:
If channels have different ownership rules from other objects it would be helpful if there were a comment explaining this somewhere. I have C code where I call `lovrThreadGetChannel` directly and later (when I'm done calling it) currently release it. I can follow whatever rules I just need to know what they are.